### DISCUSSION TITLE
Can you use a single facet list component to drive multiple result lists?

### DISCUSSION THREAD

**Discussion by tombburnell** [appbaseio/reactivesearch#1867] [Q&A]
I want to have multiple lists of results on my home page “Trending”,  “Top Sellers” "editor picks" etc..

And I’d like to have a single multi-select facet control for “Genre” so that when the user selects the genre’s they are interested in, all result lists update to show only products in those selected genres.

I'd also like to store their selections for their next visit.

Is it possible and how would you structure it in reactivesearch?

Thanks!

**Comment by mohdashraf010897** [comment]
@tombburnell It is very much possible to reactively update results of different-multiple ReactiveList (s).
Check this https://p464qg.sse.codesandbox.io/ 
This example mimics what you have asked, we have got two lists that are set to be updated to one multi-selectable facet component. 


> I'd also like to store their selections for their next visit.

I didn't quite get this, so maybe you want to explain a bit more.
